### DISCUSSION TITLE
How to customize trade open & close profit/loss margins in Octobot running strategy?

### DISCUSSION THREAD

**Discussion by mkamranmazhar** [Drakkar-Software/OctoBot#2648] [Q&A] [answered]
Hello Developer Support,

I'm currently using Octobot on my VPS and I would like to know if it's possible to customize the profit and loss margins for the currently chosen/running strategy.

For example, Octobot opens a trade at $3, and I would like it to close at $3.50. I've noticed that trades often close after only a 1 to 5 cent profit, but I want to increase this range to a minimum and maximum of $0.50 for both profit and loss.

Thank you.

Best regards,

**Comment by GuillaumeDSM** [comment, marked as answer]
Hey @mkamranmazhar,
In your trading mode configuration, you can choose how sell / exit order prices are computed. This allows you to target specific profits/stop losses
